.github/workflows/sync-packages.yaml

```
@@ -32,7 +32,13 @@ jobs:
                         --path var/spack/repos/ --path-rename var/spack/repos/:repos/ \
                         --path share/spack/gitlab/cloud_pipelines/ --path-rename share/spack/gitlab/cloud_pipelines/:.ci/gitlab/ \
                         --path var/spack/spack-repo-index.yaml --path-rename var/spack/spack-repo-index.yaml:spack-repo-index.yaml \
-                        --path lib/spack/spack/test/ --path-rename lib/spack/spack/test/:repos/spack_repo/builtin/tests/ \
+                        --path lib/spack/spack/test/__init__.py --path-rename lib/spack/spack/test/__init__.py:repos/spack_repo/builtin/tests/__init__.py \
```

```suggestion
                        --path lib/spack/spack/test/__init__.py --path-rename lib/spack/spack/test/__init__.py:tests/__init__.py \
```

I would not expose the tests as a module `spack_repo.builtin.tests`